agoradatatools/etl/transform.py

```diff
@@ -182,146 +164,105 @@ def transform_rna_seq_data(datasets: dict, adjusted_p_value_threshold: int):
     return diff_exp_data
 
 
-def transform_network(datasets: dict):
-    gene_info = datasets['gene_info']
-    networks = datasets['networks']
-
-    gene_info.rename(columns={"symbol": "hgnc_symbol"}, inplace=True)
-    gene_info = gene_info[['ensembl_gene_id', 'hgnc_symbol']]
-    gene_info.drop_duplicates(inplace=True)
-
-    networks = networks[
-        networks['genea_ensembl_gene_id'].isin(gene_info['ensembl_gene_id']) &
-        networks['geneb_ensembl_gene_id'].isin(gene_info['ensembl_gene_id'])]
-
-    merged = pd.merge(left=networks,
-                      right=gene_info,
-                      left_on='genea_ensembl_gene_id',
-                      right_on='ensembl_gene_id',
-                      how='left')
-
-    merged = pd.merge(left=networks,
-                      right=gene_info,
-                      left_on='geneb_ensembl_gene_id',
-                      right_on='ensembl_gene_id',
-                      how='left')
-    merged = merged[['genea_ensembl_gene_id', 'geneb_ensembl_gene_id',
-                     'genea_external_gene_name', 'geneb_external_gene_name', 'brainregion']]
-
-    return merged
+def fix_alias_field(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
You're right, this should be in-line. I had originally pulled it out because I intended to use it in two functions, but then ended up not needing to do that. I'll move it.

Reply:
Moved back in-line. (commit [1258ed40f91d6842e381cbaaac1eeef0d729c6cf](https://github.com/jaclynbeck-sage/agora-data-tools/commit/1258ed40f91d6842e381cbaaac1eeef0d729c6cf))